Source/GTMSessionFetcher.m

```diff
@@ -1857,6 +1857,7 @@ - (void)stopFetchReleasingCallbacks:(BOOL)shouldReleaseCallbacks {
 
   [holdSelf destroyRetryTimer];
 
+  BOOL sendStopNotification = YES;
```

Comment:
This changes doesn't seem to be part of the rest of the other changes, do you want to move this to a different commit so it isn't lost in the git history?